packages/cli/src/services/redis/RedisServiceBaseClasses.ts

```diff
@@ -40,10 +40,6 @@ class RedisServiceBase {
 		}
 		this.redisClient = this.redisClientService.createClient({ type });
 
-		this.redisClient.on('close', () => {
```

Comment:
instead of removing this, could we change the event from `close` to `reconnecting`